Servers/WindowServer/WSMenu.h

```diff
@@ -94,5 +100,7 @@ class WSMenu final : public CObject {
     WSMenuItem* m_hovered_item { nullptr };
     NonnullOwnPtrVector<WSMenuItem> m_items;
     RefPtr<WSWindow> m_menu_window;
+    WSWindow* m_window_menu_of { nullptr };
+    bool m_is_window_menu_open = false;
```

Comment:
Please use brace syntax to be consistent with the rest of the code:
```suggestion
    bool m_is_window_menu_open = false;
```